src/core_plugins/kibana/public/context/query.js

```diff
@@ -0,0 +1,154 @@
+import _ from 'lodash';
+
+import { fetchAnchor } from './api/anchor';
+import { fetchPredecessors, fetchSuccessors } from './api/context';
+import { QueryParameterActionsProvider } from './query_parameters';
+
+
+function QueryActionsProvider(es, Notifier, Private, Promise) {
+  const {
+    increasePredecessorCount,
+    increaseSuccessorCount,
+    setPredecessorCount,
+    setQueryParameters,
+    setSuccessorCount,
+  } = Private(QueryParameterActionsProvider);
```

Comment:
Are these actions needed elsewhere? What's the intention behind extracting them into their own file?